app/models/cloud9_invite.rb

```diff
@@ -0,0 +1,14 @@
+class Cloud9Invite < ApplicationRecord
+  validates :email, presence: true, uniqueness: true, email: true
+
+  before_create :invite_member
+
+  def invite_member
+    Cloud9Client::Team.invite_member(
+      Rails.application.secrets.cloud9_team_name,
+      email
+    )
+  rescue RestClient::Conflict
+    errors.add(:send_invite, "Couldn't invite member")
+  end
```

Comment:
I'd make this a private method. We don't want people manually trying to send an invite twice.

The error message added when `RestClient::Conflict` is hit should be the same as the error message returned when the uniqueness validator fails. `RestClient::Conflict` is only raised when Cloud9 has already received an invite for the given email.